manifests/config.pp

```diff
@@ -8,8 +8,22 @@
 
   ensure_resource('jenkins::plugin', $::jenkins::default_plugins)
 
+  $java_args = concat(
+    $::jenkins::params::_default_java_args,
+    $::jenkins::java_args
+  )
+
+  $config_defaults = {
+    "${::jenkins::params::config_prefix}JAVA_ARGS" => {
+      value => join($java_args, ' ')
+    },
+    "${::jenkins::params::config_prefix}AJP_PORT" => {
+      value => '-1'
```

Comment:
I really don't like to see default values hard-coded outside of the params class. Someday, hopefully soon, we'll be able to replace the params pattern with data-in-modules.